lib/entity.js

```diff
@@ -82,6 +112,20 @@ Entity.prototype.mergeProperty = function mergeProperty (name, value) {
   else this[name] = value;
 };
 
+/**
+ * Replay an array of events onto the instance.
+ *
+ * The goal here is to allow application of events without emitting, enqueueing
+ * nor digesting the replayed events. This is done by setting this.replaying to
+ * true which emit, enqueue and digest check for.
+ *
+ * - If the method in the event being replayed exists in the instance, we call
+ * the mathod with the data in the event and set the version of the instance to
+ * the version of the event.
+ * [CONFIRM] - What's happening in the else block where classNameRegex is used?
```

Comment:
If you reach that code, it means that the method that was expected to be found on the object (because some events were saved to a database and then replayed) but they were not found on the object they were being applied to. 

The regexp code is some helper code to provide descriptive errors to the developer of 1) the type of the object we're attempting to apply the method on and 2) the name of the method.